File: Applications/C++ & C# programming languague/Lab5/Lab5Tema/Lab5tema/Active.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace Lab5tema
{
    public class Active : Sport
    {
        public int years_activity;
        public int suspended;
        public void set_MatchesPlayed(int matches)
        {
            matches -= suspended;
        }

        //Lab5
        public void display_years_activity()
        {
            Console.WriteLine("Display years of activity: " + years_activity + " (called by Active Class)");
        }

        public virtual void display_suspended()
        {
            Console.WriteLine("Display days as suspended: " + suspended + " (called by Active Class)");
        }
    }
}

File: Applications/C++ & C# programming languague/Lab4/Lab4Tema - metoda2/Lab1Tema/Active.cpp
#include <iostream>
#include "Sport.h"
#include "Active.h"

using namespace std;

void Active::set_MatchesPlayed(int matches)			//Lab3ex4
{
	matches -= suspended;
	Sport::set_MatchesPlayed(matches);
}

void Active::Active_E_access()						//Lab3ex5
{
	consume.display_hydrated();
}

//Lab4

void Active::display_years_activity()
{
	cout << "Display years of activity: "<< years_activity << " (called by Active Class)" << endl;		//Lab4ex1
}

void Active::display_suspended()
{
	cout << "Display days as suspended: " << suspended << " (called by Active Class)" << endl;		//Lab4ex2
}